<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #F7FFFF;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Labe_out">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #475679;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_del">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>185</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #ff584c;
    color: #ffd8d6;
    border: 2px outset #ff4032;
}

QPushButton:pressed {
    background-color: #ff4032;
    border: 2px outset #e5392d;
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_plus_minus">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>185</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #F2B33D;
    color: white;
    border: 2px outset #d9a136;
}

QPushButton:pressed {
    background-color: #d9a136;
    border: 2px outset #c18f30;
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>325</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #F2B33D;
    color: white;
    border: 2px outset #d9a136;
}

QPushButton:pressed {
    background-color: #d9a136;
    border: 2px outset #c18f30;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>325</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #F2B33D;
    color: white;
    border: 2px outset #d9a136;
}

QPushButton:pressed {
    background-color: #d9a136;
    border: 2px outset #c18f30;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>395</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #F2B33D;
    color: white;
    border: 2px outset #d9a136;
}

QPushButton:pressed {
    background-color: #d9a136;
    border: 2px outset #c18f30;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>465</y>
      <width>130</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #F2B33D;
    color: white;
    border: 2px outset #d9a136;
}

QPushButton:pressed {
    background-color: #d9a136;
    border: 2px outset #c18f30;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>395</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #F2B33D;
    color: white;
    border: 2px outset #d9a136;
}

QPushButton:pressed {
    background-color: #d9a136;
    border: 2px outset #c18f30;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_percent">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>185</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #F2B33D;
    color: white;
    border: 2px outset #d9a136;
}

QPushButton:pressed {
    background-color: #d9a136;
    border: 2px outset #c18f30;
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>325</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #F2B33D;
    color: white;
    border: 2px outset #d9a136;
}

QPushButton:pressed {
    background-color: #d9a136;
    border: 2px outset #c18f30;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_minus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>60</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #7AA4FF;
    color: white;
    border: 2px outset #6d93e5;
}

QPushButton:pressed {
    background-color: #6d93e5;
    border: 2px outset #6183cc;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_all_del">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>185</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #b20b00;
    color: #ffb7b2;
    border: 2px outset #990a00;
}

QPushButton:pressed {
    background-color: #990a00;
    border: 2px outset #7f0800;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>395</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #F2B33D;
    color: white;
    border: 2px outset #d9a136;
}

QPushButton:pressed {
    background-color: #d9a136;
    border: 2px outset #c18f30;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_dot">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>465</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #F2B33D;
    color: white;
    border: 2px outset #d9a136;
}

QPushButton:pressed {
    background-color: #d9a136;
    border: 2px outset #c18f30;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_equals">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>465</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #C1E80C;
    color: white;
    border: 2px outset #add00a;
}

QPushButton:pressed {
    background-color: #add00a;
    border: 2px outset #9ab909;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_plus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>413</y>
      <width>60</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #7AA4FF;
    color: white;
    border: 2px outset #6d93e5;
}

QPushButton:pressed {
    background-color: #6d93e5;
    border: 2px outset #6183cc;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>255</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #F2B33D;
    color: white;
    border: 2px outset #d9a136;
}

QPushButton:pressed {
    background-color: #d9a136;
    border: 2px outset #c18f30;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_mult">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>308</y>
      <width>60</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #7AA4FF;
    color: white;
    border: 2px outset #6d93e5;
}

QPushButton:pressed {
    background-color: #6d93e5;
    border: 2px outset #6183cc;
}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>255</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #F2B33D;
    color: white;
    border: 2px outset #d9a136;
}

QPushButton:pressed {
    background-color: #d9a136;
    border: 2px outset #c18f30;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>255</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #F2B33D;
    color: white;
    border: 2px outset #d9a136;
}

QPushButton:pressed {
    background-color: #d9a136;
    border: 2px outset #c18f30;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_cos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 9px;
    background-color: #5572b2;
    color: white;
    border: 2px outset #496299;
}

QPushButton:pressed {
    background-color: #496299;
    border: 2px outset #3d527f;
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_sin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>115</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 9px;
    background-color: #5572b2;
    color: white;
    border: 2px outset #496299;
}

QPushButton:pressed {
    background-color: #496299;
    border: 2px outset #3d527f;
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_tan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 9px;
    background-color: #5572b2;
    color: white;
    border: 2px outset #496299;
}

QPushButton:pressed {
    background-color: #496299;
    border: 2px outset #3d527f;
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_acos">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 9px;
    background-color: #5572b2;
    color: white;
    border: 2px outset #496299;
}

QPushButton:pressed {
    background-color: #496299;
    border: 2px outset #3d527f;
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_asin">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>115</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 9px;
    background-color: #5572b2;
    color: white;
    border: 2px outset #496299;
}

QPushButton:pressed {
    background-color: #496299;
    border: 2px outset #3d527f;
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_atan">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 9px;
    background-color: #5572b2;
    color: white;
    border: 2px outset #496299;
}

QPushButton:pressed {
    background-color: #496299;
    border: 2px outset #3d527f;
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_exp">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>115</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 9px;
    background-color: #5572b2;
    color: white;
    border: 2px outset #496299;
}

QPushButton:pressed {
    background-color: #496299;
    border: 2px outset #3d527f;
}</string>
    </property>
    <property name="text">
     <string>a^b</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_ln">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 9px;
    background-color: #5572b2;
    color: white;
    border: 2px outset #496299;
}

QPushButton:pressed {
    background-color: #496299;
    border: 2px outset #3d527f;
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_log">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 9px;
    background-color: #5572b2;
    color: white;
    border: 2px outset #496299;
}

QPushButton:pressed {
    background-color: #496299;
    border: 2px outset #3d527f;
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>115</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 9px;
    background-color: #5572b2;
    color: white;
    border: 2px outset #496299;
}

QPushButton:pressed {
    background-color: #496299;
    border: 2px outset #3d527f;
}</string>
    </property>
    <property name="text">
     <string>√a</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_percent_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 9px;
    background-color: #5572b2;
    color: white;
    border: 2px outset #496299;
}

QPushButton:pressed {
    background-color: #496299;
    border: 2px outset #3d527f;
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_21">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 9px;
    background-color: #5572b2;
    color: white;
    border: 2px outset #496299;
}

QPushButton:pressed {
    background-color: #496299;
    border: 2px outset #3d527f;
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
    background-color: rgba(0, 0, 0, 0.08);</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
   <widget class="QPushButton" name="PushButton_div">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>255</y>
      <width>60</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #7AA4FF;
    color: white;
    border: 2px outset #6d93e5;
}

QPushButton:pressed {
    background-color: #6d93e5;
    border: 2px outset #6183cc;
}</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
